src/main.c

```diff
@@ -85,6 +87,7 @@ int get_key(void) {
 void press_exit(void) {
 	printf("Press any key to exit this application.\n");
 	get_key();
+	sceKernelExitProcess(0);
```

Comment:
Is sceKernelExitProcess() and exit() different? Maybe I should remodify the code.